source/slang/slang-ast-builder.h

```diff
@@ -529,9 +526,30 @@ class ASTBuilder : public RefObject
     SharedASTBuilder* m_sharedASTBuilder;
 
     MemoryArena m_arena;
+};
+
+// Retrieves the ASTBuilder for the current compilation session.
+ASTBuilder* getCurrentASTBuilder();
```

Comment:
I'll be interested to see if this is handled via a thread-local, or what...

Reply:
It's a thread_local.